crates/wasm_interp/src/instance.rs

```diff
@@ -53,20 +64,18 @@ impl<'a> Instance<'a> {
             block_loop_addrs: Vec::new_in(arena),
             outermost_block: 0,
             import_signatures: Vec::new_in(arena),
+            import_dispatcher,
+            import_arguments: Vec::new_in(arena),
             debug_string: Some(String::new()),
         }
     }
 
-    pub fn for_module<'arg, A>(
+    pub fn for_module(
         arena: &'a Bump,
         module: &WasmModule<'a>,
-        start_fn_name: &str,
```

Comment:
Previously `start_fn_name` would specify _on initialisation_ which function you're going to execute. But that's not a great API! There was no method to run a different exported function later, or to make multiple calls.
In this new version, `for_module` just initialises things, and you can call `call_export` as many times as you like, on different exports.